### ISSUE TITLE
Invalid Iris dependency

### ISSUE THREAD

**Issue by adam-debkowski** [awslabs/aws-lambda-go-api-proxy#70]
Since today I'm unable to build my go application using `aws-lambda-go-api-proxy` due to an invalid version of the `iris` dependency.

```
go test ./... -covermode=count -count=1
go: github.com/awslabs/aws-lambda-go-api-proxy@v0.7.0 requires
	github.com/kataras/iris@v11.1.1+incompatible: reading github.com/kataras/iris/go.mod at revision v11.1.1: unknown revision v11.1.1
```

And after looking into the `github.com/kataras/iris`, there is no `v11.1.1` version. At this point, I'm not sure how did it work before - did the `github.com/kataras/iris@v11.1.1` got deleted from their github, or am I missing something?

**Comment by sapessi** [comment]
I just tried to run the same command on my machine and it worked. I have attempted to clean the ~/go/pkg/mod cache and every time I see the `iris@v11.1.1+incompatible` folder reappear. Weirdly, I cannot see the v11 tag in the iris repo either.

**Comment by mescam** [comment]
@sapessi 
It's because you are downloading cached version from gopkg.

The maintainer of iris said that he deleted all of legacy tags: https://github.com/kataras/iris/issues/1571

**Comment by kataras** [comment]
Iris users were not the only ones left behind by using legacy package import (github.com/kataras/iris instead of github.com/kataras/irisv12). The users of echo web framework, was left behind too. Echo's latest version is `v4` (github.com/labstack/echo/v4) but this package requires the `github.com/labstack/echo` (which was version 3).

So, I am preparing a PR which will update all dependencies, including Iris and Echo.

**Comment by kataras** [comment]
OK It's done, it was a simple task. Please vote up or comment your opinion at: https://github.com/awslabs/aws-lambda-go-api-proxy/pull/71.

Thanks,
Gerasimos Maropoulos.

**Comment by sapessi** [comment]
I'm working on the iris update now. Echo is a bigger breaking change. I think we should just switch to #65 and simply support the http adapter. We can then give samples that show how to use it with each framework.

**Comment by sapessi** [comment]
See #72 - I may merge and tag as a major release

**Comment by sapessi** [comment]
Closing since we merged #71
